openeo/capabilities.py

```diff
@@ -52,7 +51,33 @@ def list_plans(self):
 
 class ComparableVersion:
     """
-    Helper to compare a version (e.g. API version) against another (threshold) version
+    This code is copied from distutils.version.LooseVersion
```

Comment:
Now after being merged, I would not say "This code is copied", but "implementation is based on"

Reply:
done